cloudformation/cloudformation-sns-emission-consumer.yml

```diff
@@ -30,106 +62,106 @@ Resources:
-
-                # Force resources in stacks to only be able to update items that they
-                # created
-                item[ACCOUNT_ID_KEY] = message['StackId'].split(':')[4]
-                item.setdefault(CATEGORY_KEY, GENERAL_ITEM_CATEGORY)
-                item[SORT_KEY] = '{}+{}'.format(
-                    stack_guid,
-                    message['LogicalResourceId'])
-
-                item[STACK_ID_KEY] = stack_guid
-                item[LOGICAL_RESOURCE_ID_KEY] = message['LogicalResourceId']
-
-                item.setdefault('stack-name', stack_path.split('/')[1])
-                item.setdefault('region', message['StackId'].split(':')[3])
-                item.setdefault(LAST_UPDATED_KEY, datetime.utcnow().isoformat() + 'Z')
-
-                dynamodb = boto3.resource('dynamodb', region_name=TABLE_REGION)
-
-                if message['RequestType'] == 'Delete':
-                    table = dynamodb.Table(TABLE_NAME)
-                    table.delete_item(
-                        Key={ACCOUNT_ID_KEY: item[ACCOUNT_ID_KEY],
-                             SORT_KEY: item[SORT_KEY]})
-                    # We don't check to see if the table is now empty and can be deleted
-                    # because there's no cheap or easy way to determine if a table is empty
-                    # using either ItemCount or Scan
-                elif message['RequestType'] in ['Create', 'Update']:
-                    table = dynamodb.Table(TABLE_NAME)
-                    table.put_item(Item=item)
-
-
-            def handler(event, context):
-                message = always_succeed = None
-                try:
-                    for record in event['Records']:
-                        message = json.loads(record['Sns']['Message'])
-                        always_succeed = message['RequestType'] == 'Delete'
-                        update_table(message)
-                except Exception:
-                    print('Custom resource failed. Exception: {0}\n{1}'.format(
-                        traceback.format_exc(), event))
-                    status = cfnresponse.SUCCESS if always_succeed else cfnresponse.FAILED
-                else:
-                    print('Custom resource succeeded.')
-                    status = cfnresponse.SUCCESS
-                if 'PhysicalResourceId' in message:
-                    physical_id = message['PhysicalResourceId']
-                else:
-                    random_string = ''.join(
-                        secrets.choice(string.ascii_uppercase + string.digits)
-                        for _ in range(13))
-                    physical_id = "ProcessCloudFormationSNSEmission-{}".format(
-                        random_string)
-                cfnresponse.send(message, context, status, {}, physical_id)
-          - DynamoDBTableName: !FindInMap [ Variables, DynamoDBTable, Name ]
-            DynamoDBTableRegion: !FindInMap [ Variables, DynamoDBTable, Region ]
+        ZipFile: !Sub >
```

Comment:
This line, with the `>` breaks the code as it tells YAML to "fold" lines, removing carriage returns.